script/deploy/DeployL2PrizePool.s.sol

```diff
@@ -25,27 +25,17 @@ import { ERC20, YieldVaultMintRate } from "../../src/YieldVaultMintRate.sol";
 
 import { Helpers } from "../helpers/Helpers.sol";
 
-import { 
-    Constants,
-    DRAW_PERIOD_SECONDS,
-    GRAND_PRIZE_PERIOD_DRAWS,
-    TIER_SHARES,
-    RESERVE_SHARES,
-    AUCTION_DURATION,
-    TWAB_PERIOD_LENGTH,
-    AUCTION_TARGET_SALE_TIME,
-    CLAIMER_MAX_FEE,
-    CLAIMER_MIN_FEE,
-    ERC5164_EXECUTOR_GOERLI_OPTIMISM
-} from "./Constants.sol";
+import { Constants, DRAW_PERIOD_SECONDS, GRAND_PRIZE_PERIOD_DRAWS, TIER_SHARES, RESERVE_SHARES, AUCTION_DURATION, TWAB_PERIOD_LENGTH, AUCTION_TARGET_SALE_TIME, CLAIMER_MAX_FEE, CLAIMER_MIN_FEE, ERC5164_EXECUTOR_GOERLI_OPTIMISM } from "./Constants.sol";
```

Comment:
I don't agree with this formatting

Reply:
I don't what happened but that's how `npm run format` formatted it.
We can probably update the prettier config.